tools/run_tests/helper_scripts/build_csharp.bat

```diff
@@ -31,9 +31,7 @@ setlocal
 
 cd /d %~dp0\..\..\..\src\csharp
 
-dotnet restore . || goto :error
-
-dotnet build --configuration %MSBUILD_CONFIG% "**/project.json" || goto :error
+dotnet build --configuration %MSBUILD_CONFIG% Grpc.sln || goto :error
```

Comment:
gcov doesn't apply for windows in terms of C core coverage.
We use it for C# code coverage, but for that, one doesn't need to override the version of grpc_csharp_ext being used. Hence the difference between linux and windows.